pkg/cmd/backuprestore/crioclient.go

```diff
@@ -0,0 +1,117 @@
+package backuprestore
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"k8s.io/klog"
+	"net"
+	"net/url"
+	"time"
+
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+const (
+	defaultTimeout = 2 * time.Second
+	unixProtocol   = "unix"
+)
+
+// Code taken from github.com/kubernetes-sigs/cri-tools/cmd/crictl
+
+var defaultRuntimeEndpoint string = "unix:///run/crio/crio.sock"
+
+func getRuntimeCrioClient() (pb.RuntimeServiceClient, *grpc.ClientConn, error) {
+	// Set up a connection to the server.
+	conn, err := getRuntimeClientConnection()
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "connect")
```

Comment:
We shouldn't be dragging in this third party `errors` package since the stdlib already supports wrapping through `fmt.Errorf`